_blog/2018/aws-nlb.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Configure Istio Ingress with AWS NLB"
+overview: Describes how to configure istio ingress with an external load balancer NLB on AWS
+publish_date: April 08, 2018
+subtitle: Ingress AWS NLB
+attribution: Julien SENON
+
+order: 94
+
+
+layout: blog
+type: markdown
+redirect_from: "/blog/aws-nlb.html"
+---
+{% include home.html %}
+
+Instructions to use and configure ingress istio with `aws nlb`.
+
+## Prerequisites
+
+The following instructions require you have access to a Kubernetes **1.9.0 or newer** cluster.
+
+<img src="{{home}}/img/exclamation-mark.svg" alt="Warning" title="Warning" style="width: 32px; display:inline" />  This is an alpha feature and not recommended for production clusters.
+
```

Comment:
Might add an introduction as to what is needed at a high level and what the objectives are of the blog post.